### PR TITLE
[MIG] product_code_unique: Migration to 13.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

File: product_code_unique/tests/test_code_unique.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import psycopg2
+from odoo.tests.common import SavepointCase
+from odoo.tools.misc import mute_logger
+
+
+class TestCodeUnique(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestCodeUnique, cls).setUpClass()
```
Comment (i-vyshnevska) [inline product_code_unique/tests/test_code_unique.py]:
```suggestion
        super().setUpClass()
```

**Review by i-vyshnevska — APPROVED** [review]
LG

**Review by schout-it — APPROVED** [review]
lgtm

File: product_code_unique/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+* Antonio Yamuta <ayamuta@opensourceintegrators.com>
+* Raf Ven <raf.ven@dynapps.be>
+
+Funders
```
Comment (pedrobaeza) [inline product_code_unique/readme/CONTRIBUTORS.rst]: Remove this part, as they are already creators.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: product_code_unique/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (rven) [reply product_code_unique/readme/CONTRIBUTORS.rst]: > Remove this part, as they are already creators.

Do you mean the Funders parts?

Comment (pedrobaeza) [reply product_code_unique/readme/CONTRIBUTORS.rst]: Yes

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**